Translations/translation_PL.json

```diff
@@ -31,12 +31,12 @@
 			""
 		],
 		"NoAccelerometerMessage": [
-			"No accelerometer",
-			"detected!"
+			"Nie rozpoznano",
+			"akcelerometru!"
 		],
 		"NoPowerDeliveryMessage": [
-			"No USB-PD IC",
-			"detected!"
+			"Nie rozpoznano",
+			"kontrolera USB-PD IC!"
```

Comment:
Unfortunately this line is too long to fit on the screen. The limit is 16 chars per line.

Reply:
Ok, I shortened the word